bot/seasons/evergreen/bookmark.py

```diff
@@ -46,6 +63,25 @@ def __init__(self, bot: commands.Bot):
         else:
             await ctx.message.add_reaction(Emojis.envelope)
 
+        embed.add_field(
+            name=f'Bookmarked from {ctx.author.name}.',
+            value=f'[Visit original message]({ctx.message.jump_url})',
+            inline=False
+        )

```

Comment:
What is the purpose of this additional field? Doesn't it just duplicate the same information already in the embed with a different field name?